### DISCUSSION TITLE
QField (2.3.4): Set bookmarks are not stored

### DISCUSSION THREAD

**Discussion by iwaend-gibs** [opengisch/QField#3442] [Bugs]
Hi,
using android 11 and QField 2.3.4, set bookmarks are displayed but not stored and disappear when the project is closed and re-opened. In the settings, bookmarks are set to visible. Are the bookmarks lost? Is there a possibility to check on any files?

Cool project by the way. 

Kind regards
Yven

**Comment by nirvn** [comment]
@iwaend-gibs , can you give QField 2.4.1 a try?

**Comment by iwaend-gibs** [comment]
Hi nirvn,
I installed "qfield-v2.4.1-arm64-android.apk". The previous generated project worked but showed the same behaviour: creating spatial bookmarks works, paning away and back: still there. But after closing and re-opening, the bookmarks are gone.
Second test: Two test bookmarks set in the example project "bees" - switched to my project without shutting down QField and then back to the bee project. The test bookmarks in the bee project remain still there. After closing (shutting down) QField and re-opening it, the bookmarks are gone.
Thanks a lot for taking care!
Cheers
Yven
